perfdash/config.go

```diff
@@ -79,6 +79,11 @@ var (
 			OutputFilePrefix: "APIResponsiveness",
 			Parser:           parseRequestCountData,
 		},
+		"DensityApiserverRequestCount": {
```

Comment:
`DensityRequestCount` should be enough. Same below.

Reply:
Ok, seems like that's already taken. How about `DensityRequestCountByClient`?

Reply:
Done